metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -320,6 +320,16 @@ class TableauConfig(
         "By default, all projects will be ingested.",
     )
 
+    project_path_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Filter for specific Tableau projects by checking their full path. For example, use 'My Project/Nested Project' to ingest a nested project with name 'Nested Project'. "
```

Comment:
Could you please update the description as per below text:

"Filters Tableau projects by their full path. For instance, 'My Project/Nested Project' targets a specific nested project named 'Nested Project'."
" Unlike project_pattern, this field only checks the project path, not both the path and project name."
" This is useful when you need to exclude all nested projects under a particular project."
" You can allow or deny projects by specifying their path or a regular expression pattern."
" Deny patterns always override allow patterns."
" By default, all projects are ingested."

Reply:
Sure, changed it.